.github/workflows/sync_docs.yaml

```diff
@@ -0,0 +1,29 @@
+name: Sync docs from Discourse
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '53 0 * * *' # Daily at 00:53 UTC
+  # Triggered on push to branch "main" by .github/workflows/release.yaml
+  workflow_call:
+
+jobs:
+  sync-docs:
+    name: Open PR with docs changes
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Open PR with docs changes
+        uses: deusebio/discourse-gatekeeper@main
+        id: docs-pr
+        with:
+          discourse_host: discourse.charmhub.io
+          discourse_api_username: ${{ secrets.DISCOURSE_API_USERNAME }}
+          discourse_api_key: ${{ secrets.DISCOURSE_API_KEY }}
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          dry_run: "true"
+
+      - name: Show migrate output
+        run: echo '${{ steps.docs-pr.outputs.migrate }}'
+      - name: Show reconcile output
+        run: echo '${{ steps.docs-pr.outputs.reconcile }}'
```

Comment:
I am a noob @deusebio and @dmitry-ratushnyy - where does it get the docs it pulls from? From the [docs field in metadata](https://github.com/canonical/mongodb-k8s-operator/blob/3321534cd9baadd8e073cb607205bee13a2ba71b/metadata.yaml#L12)?

Reply:
Yes, indeed. From the topic in Disocurse, it parses the Navigation table and therefore also fetches all the other topics in the documentation, creating the corresponding files. 

Note that if something does not exists in the Navigation Table, it won't exist in the docs folder